core-bundle/src/Resources/contao/library/Contao/Controller.php

```diff
@@ -175,6 +175,14 @@ public static function getTemplateGroup($strPrefix, array $arrAdditionalMapper=a
 			$strGlobPrefix = substr($strGlobPrefix, 0, -1) . '[_-]';
 		}
 
+		// Add the matching bundle templates, e.g. in "override all" mode (see #5131)
+		$arrBundleMatches = array_unique(preg_grep('/' . $strGlobPrefix . '/', $arrBundleTemplates));
```

Comment:
@ausi Not sure if there is a better way to match the template names against the glob pattern? (Or to convert the glob pattern into a regular expression?)